adcc/adc_pp/transition_dm.py

```diff
@@ -69,7 +69,7 @@ def tdm_cvs_adc2(mp, amplitude, intermediates):
     )
 
     # cvs_adc2_dp0_vc
-    dm.vc = u1.transpose() - einsum("ab,Ib->aI", p0.vv, u1)
+    dm.vc -= 0.5 * einsum("ab,Ib->aI", p0.vv, u1)
```

Comment:
Is `u1` accounted for by the lower-order tdm?

Reply:
Exactly. In line 57, the zero-order contribution is already calculated, but in the previous implementation, this was then not used, but recalculated in line 72.